client/src/layouts/Events/RegisterEventForm/index.tsx

```diff
@@ -0,0 +1,396 @@
+        id: "ND"
+    },
+    {
+        title: "MP",
+        id: "MP"
+    },
+    {
+        title: "OH",
+        id: "OH"
+    },
+    {
+        title: "OK",
+        id: "OK"
+    },
+    {
+        title: "OR",
+        id: "OR"
+    },
+    {
+        title: "PW",
+        id: "PW"
+    },
+    {
+        title: "PA",
+        id: "PA"
+    },
+    {
+        title: "PR",
+        id: "PR"
+    },
+    {
+        title: "RI",
+        id: "RI"
+    },
+    {
+        title: "SC",
+        id: "SC"
+    },
+    {
+        title: "SD",
+        id: "SD"
+    },
+    {
+        title: "TN",
+        id: "TN"
+    },
+    {
+        title: "TX",
+        id: "TX"
+    },
+    {
+        title: "UT",
+        id: "UT"
+    },
+    {
+        title: "VT",
+        id: "VT"
+    },
+    {
+        title: "VI",
+        id: "VI"
+    },
+    {
+        title: "VA",
+        id: "VA"
+    },
+    {
+        title: "WA",
+        id: "WA"
+    },
+    {
+        title: "WV",
+        id: "WV"
+    },
+    {
+        title: "WI",
+        id: "WI"
+    },
+    {
+        title: "WY",
+        id: "WY"
+    }
+]
+
+const initValues:FormFields = { 
+    name: '',
+    category: '',
+    date: '',
+    starttime: '',
+    endtime: '',
+    description: '',
+    provider: '',
+    url: '',
+    location: '',
+    street: '',
+    city: '',
+    state: '',
+    zipcode: '',
+}
+
+export default function RegisterEventForm(): React.ReactElement {
+
+    const [img64Base, setImg64Base] = React.useState<string | undefined>(undefined)
+    //const [inputFileRef, setInputFileRef] = React.useState<React.RefObject<HTMLInputElement> | null>(null)
+
+    const [wholeDayEvent, setWholeDayEvent] = useState(false)
+    const [onlineEvent, setOnlineEvent] = useState(false)
+
+    const methods = useForm<FormFields>({
+        defaultValues: initValues,
+        resolver: yupResolver(EventsSchema), 
+        mode: "onChange"
+    });
+
+    const onSubmit = (data:FormFields) => {
+
+        Swal.fire({
+            title: "We have received your event!",
+            text: "After our team reviews the information, we'll make sure your event gets the visibility it deserves.",
+            icon: "success"
+        });
+
+        console.log(data)
+    }
+    
+    const handleWholeDayEvent = () => setWholeDayEvent(!wholeDayEvent)
+    const handleOnlineEvent = () => setOnlineEvent(!onlineEvent)
+
+    return (
+        <article className={styles.formContainer}>
+            <FormProvider {...methods}>
+                <form className={styles.eventForm} onSubmit={methods.handleSubmit(onSubmit)}>
+                    <div className={styles.eventDetailsContainer}>
+                        <EventFormTitle title="Event Details" />
+                        <TextFieldBase id="name" title='Event name*' placeholder="Event name" fullwidth={true} />
+                        <TextFieldBase id="provider" title='Event provider*' placeholder="Event provider" fullwidth={true} />
```

Comment:
I would prefer this as **Event Title** instead of **Event Name**
and **Organization Name** instead of **Event Provider**. 
but let me know what you think.